### ISSUE TITLE
Fullscreen Flash locks up when you maximize an MDIWindow in Chrome

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#2437]
Originally reported on Google Code with ID 1685
```
What steps will reproduce the problem?
1. Open the Flash client in Chrome
2. Press F11 or make the browser full screen
3. Maximize an MDIWindow
4. The screen locks up and can't be interacted with

I can only seem to reproduce the problem in single monitor setups. On my 2 monitor
workstation the problem doesn't occur.

```

Reported by `capilkey` on 2013-11-14 18:56:04

**Comment by capilkey** [comment]
I am still running into this issue in the 1.0-beta client. I'm going to reopen the issue just to keep track of it. 

@GhaziTriki Do you happen to remember which PR was proposed to have fixed this issue? I tried to search and find it, but I had no luck.

The stacktrace is the same as the one in a previous comment except for a newer SDK version.
Error: Error #1502: A script has executed for longer than the default timeout period of 15 seconds.
	at mx.core::UIComponent/processEffectFinished()[/Users/aharui/release4.13.0/frameworks/projects/framework/src/mx/core/UIComponent.as:12396]
	at mx.core::UIComponent/updateCompleteHandler()[/Users/aharui/release4.13.0/frameworks/projects/framework/src/mx/core/UIComponent.as:12368]
	at flash.events::EventDispatcher/dispatchEventFunction()
	at flash.events::EventDispatcher/dispatchEvent()
	at mx.managers::LayoutManager/doPhasedInstantiation()[/Users/aharui/release4.13.0/frameworks/projects/framework/src/mx/managers/LayoutManager.as:859]
	at mx.managers::LayoutManager/validateNow()[/Users/aharui/release4.13.0/frameworks/projects/framework/src/mx/managers/LayoutManager.as:886]
	at mx.effects.effectClasses::TweenEffectInstance/onTweenEnd()[/Users/aharui/release4.13.0/frameworks/projects/framework/src/mx/effects/effectClasses/TweenEffectInstance.as:438]
	at mx.effects.effectClasses::ResizeInstance/onTweenEnd()[/Users/aharui/release4.13.0/frameworks/projects/mx/src/mx/effects/effectClasses/ResizeInstance.as:503]
	at mx.effects::Tween/endTween()[/Users/aharui/release4.13.0/frameworks/projects/framework/src/mx/effects/Tween.as:577]
	at mx.effects::Tween/http://www.adobe.com/2006/flex/mx/internal::doInterval()[/Users/aharui/release4.13.0/frameworks/projects/framework/src/mx/effects/Tween.as:620]
	at mx.effects::Tween$/timerHandler()[/Users/aharui/release4.13.0/frameworks/projects/framework/src/mx/effects/Tween.as:202]
	at flash.utils::Timer/_timerDispatch()
	at flash.utils::Timer/tick()

**Comment by GhaziTriki** [comment]
@capilkey it was this one #385

**Comment by GhaziTriki** [comment]
I am not able to reproduce this issue. @capilkey can you provide exact steps to reproduce it and more information about your environment (browser, fp version, os...)? And if you are in debug mode, can you please tell us what component is triggering this error?

**Comment by capilkey** [comment]
1. Load the client
2. Maximize the Presentation Window
3. The whole browser is unresponsive

The only difference in steps, from the first comment to this one, is that maximizing the browser first is not required and It was happening on my dual monitor computer that I couldn't reproduce the issue on the first time.

My setup was Win7, FF44, Flash 20.0.0.286. And @ffdixon was experiencing it as well with the Win10, FF44, Flash 20.

I'm not sure what you mean by which component is triggering the error. It's an error that is thrown without any other intervention. Roughly 15s after maximizing the window the error pops up because that is the Flash execution time limit.

Another thing to note is that both FF and Flash were maxing out a full core of CPU each while the lock up was going on.

**Comment by capilkey** [comment]
@GhaziTriki: I might have caused this to break again. In #385 you change the MainCanvas height to a manually calculated value, but the blame for MainAplicationShell.mxml shows that I changed height to "100%" again, https://github.com/bigbluebutton/bigbluebutton/blame/master/bigbluebutton-client/src/org/bigbluebutton/main/views/MainApplicationShell.mxml#L667. I had removed the listener that you were hooking into for the browser resize event so I removed your calculation as well. I'll put the calculation back in and see how it works out.
